rules-tests/Php55/Rector/FuncCall/PregReplaceEModifierRector/Fixture/skip_curly_parentheses_delimiter.php.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Rector\Tests\Php55\Rector\FuncCall\PregReplaceEModifierRector\Fixture;
+
+// https://3v4l.org/fqdPZ#v5.4.45 vs https://3v4l.org/Zm16C#v5.5.38
+function clean_link($a)
+{
+    return "change";
+}
+
+function run($str)
+{
+    $eval_links = '$link = clean_link("$0")';
+
+    return preg_replace('{test}ei',"$eval_links",$str);
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php55\Rector\FuncCall\PregReplaceEModifierRector\Fixture;
+
+// https://3v4l.org/fqdPZ#v5.4.45 vs https://3v4l.org/Zm16C#v5.5.38
+function clean_link($a)
+{
+    return "change";
+}
+
+function run($str)
+{
+    $eval_links = '$link = clean_link("$0")';
+
+    return preg_replace_callback('{tst}i',function ($matches) use ($eval_links) {
```

Comment:
oh, `e` inside regex should not be removed

Reply:
```diff
-    return preg_replace_callback('{test}i',function ($matches) use ($eval_links) {
+    return preg_replace_callback('{tst}i',function ($matches) use ($eval_links) {
         return $link = clean_link($matches[0]);
     },$str);
```
The `e` part of is a regex value itself.

Reply:
Fixed 🎉 https://github.com/rectorphp/rector-src/pull/3144/commits/652ceed5864929e87aeb5ff3ca1b1950c5c47c01